sssom/cli.py

```diff
@@ -116,7 +102,6 @@ def convert(input: str, output: str, output_format: str):
         None.
 
     """
-
```

Comment:
this should have been caught by flake8...

Reply:
Anything we can do something about, or just a comment to self?

Reply:
It might have been from a plug-in i didn’t  add yet